rdebej/encode.py

```
@@ -173,6 +174,13 @@ def get_num_bytes_and_padding(value):
 
     return num_bytes_for_value, is_padding_required
 
+def bej_pack_v_integer_fixed_bytes(stream, value, num_bytes_for_value):
+    # pack the value with the fixed length.
+    # Fixed length integer doesn't have the padding zero byte for the MSB of value set to 1.
+    # That is the consumer's prerequisite knowdlege of the value characteristic.
+    bytes = value.to_bytes(8, 'little')
+    num_bytes_packed = stream.write(bytes[:num_bytes_for_value])
```

Should throw an exception if a value is getting truncated by being reduced in length to not enough bytes